README.md

```diff
@@ -21,23 +21,27 @@ The extension is based on Tomas Votruba's [blog series](https://tomasvotruba.com
 - If you use some non-standard and magic using rendering you can implement your own template resolvers (experimental)
 
 ## Installation
+
+To use this extension, require it in [Composer](https://getcomposer.org/):
+
 ```shell
 composer require efabrica/phpstan-latte --dev
 ```
 
+If you also install [phpstan/extension-installer](https://github.com/phpstan/extension-installer) then you're all set!
+
+<details>
+  <summary>Manual installation</summary>
+
 Add this line to your phpstan.neon:
 ```neon
 includes:
     - vendor/efabrica/phpstan-latte/rules.neon
 ```
 
+</details>
+
 It is recommended to use this extension with [phpstan-nette](https://github.com/phpstan/phpstan-nette) extension for better analysis of templates.
-Add also these lines to your phpstan.neon:
```

Comment:
Do we want to remove this? User have to go to phpstan-nette and read its docs...

Reply:
I think we should not repeat docs of another extensions.